ttyg_evaluation/tools_calls_comparison.py

```diff
@@ -5,47 +5,46 @@
 
 
 def compare_tools_outputs(
-        expected_tool: dict,
-        actual_tool: dict
+        reference: dict,
+        actual: dict
 ) -> bool:
-    if "output_media_type" in expected_tool:
-        if expected_tool["output_media_type"] in {"application/sparql-results+json", "application/json"}:
-            expected_tool_output = json.loads(expected_tool["output"])
-            actual_tool_output = json.loads(actual_tool["output"])
-            if expected_tool["output_media_type"] == "application/sparql-results+json":
+    if "output_media_type" in reference:
+        if reference["output_media_type"] in {"application/sparql-results+json", "application/json"}:
+            reference_output_str = json.loads(reference["output"])
+            actual_output_str = json.loads(actual["output"])
```

Comment:
json.loads deserializes a string to a Python object, I don't know why this and the one above have suffix _str, they are not strings

Reply:
You are right! Fixed!